goblin/app.py

```diff
@@ -40,15 +44,22 @@ def __init__(self, cluster, *, provider=provider.TinkerGraph, get_hashable_id=No
         self._aliases = aliases
 
     @classmethod
-    async def open(cls, loop, *, provider=provider.TinkerGraph,
-                   get_hashable_id=None, aliases=None, **config):
+    async def open(cls,
```

Comment:
Again, really what is the difference other than personal preference.

Reply:
it's not my preference, but automatic code formatting (see `yapf`). I would like to take the personal preference out of table